client/src/view/global-fabric-panel.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2019. Beautiful Code BV, Rotterdam, Netherlands
+ * Licensed under GNU GENERAL PUBLIC LICENSE Version 3.
+ */
+
+import * as React from "react"
+import { useState } from "react"
+import {
+    FaBolt,
+    FaCompressArrowsAlt,
+    FaDownload,
+    FaParachuteBox,
+    FaRecycle,
+    FaRegFolder,
+    FaRegFolderOpen,
+    FaStarOfDavid,
```

Comment:
"Turn the fabric Jewish"

Reply:
Haha.. looking and looking for a symbol that represents merging two triangles.  Well.. I found it!